processing/src/main/java/io/druid/query/aggregation/FilteredAggregatorFactory.java

```diff
@@ -256,8 +269,20 @@ public boolean matches()
       };
     }
 
-    @Override
-    public ValueMatcher makeValueMatcher(final String dimension, final Predicate predicate)
+    public ValueMatcher makeValueMatcher(final String dimension, final DruidPredicateFactory predicateFactory)
```

Comment:
Similar comment to QueryableIndexStorageAdapter, this should probably act like the column is a string column full of nulls.